.github/workflows/humble-binary-build.yml

```diff
@@ -1,11 +1,5 @@
 name: Humble Binary Build
 on:
-  pull_request:
-    branches:
-      - main
-  push:
-    branches:
-      - main
```

Comment:
Why do we no longer want to test on these? This is inconsistent with foxy and galactic which don't have this disabled

Reply:
Oh never mind, i looked at the wrong branch. This is correct, as we only run the scheduled cis on main (apart from rolling) and the `pull_request`/`push` ones on their respective branches